src/App.tsx

```diff
@@ -1,61 +1,105 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+
+const getUserById = (userId: number) => {
+  return usersFromServer.find(user => user.id === userId) || null;
+};
+
+const listTodos = todosFromServer.map(todo => ({
+  ...todo,
```

Comment:
There is a syntax error here. The object should be constructed with key-value pairs. It seems like you intended to use 'todo' as a key, but it's missing a key name. It should be something like `{ keyName: todo }`.